core/src/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -93,6 +93,9 @@ internal MsalAccessTokenCacheItem
         [DataMember(Name = "access_token_type")]
         internal string TokenType { get; set; }
 
+        [DataMember(Name = "is_adfs")]
```

Comment:
unified schema does not contains this field, account property authority_type should be used instead

Reply:
Are you suggesting I just store the authority_type here?